sign_node/signer.py

```diff
@@ -114,61 +132,131 @@ def on_sync_request(self, queue, message):
             )
             sig_type = defaultdict(
                 lambda: '--detach-sign',
-                **{'clear-sign': '--clear-sign',
-                   'detach-sign': '--detach-sign'})
+                **{
+                    'clear-sign': '--clear-sign',
+                    'detach-sign': '--detach-sign',
+                }
+            )
             with tempfile.NamedTemporaryFile(mode='w') as fd:
-                asc_file_name = f'{fd.name}.asc'
+                asc_file_name = Path(fd.name).with_suffix('.asc')
                 fd.write(payload['content'])
                 fd.flush()
                 sign_cmd = plumbum.local['gpg'][
-                    '--yes', sig_type[payload.get('sig_type', 'detach-sign')],
-                    '--armor', '--default-key', payload['key_id'], fd.name
+                    '--yes',
+                    sig_type[payload.get('sig_type')],
```

Comment:
why we don't use `detach-sign` as default value in `get`?

Reply:
Done